cygnus-common/src/main/java/com/telefonica/iot/cygnus/metrics/CygnusMetrics.java

```diff
@@ -0,0 +1,379 @@
+            return incomingTransactionResponseSize;
+        } // getIncomingTransactionResponseSize
+
+        public long getIncomingTransactionErrors() {
+            return incomingTransactionErrors;
+        } // getIncomingTransactionErrors
+        
+        public double getServiceTime() {
+            return serviceTime;
+        } // getServiceTime
+
+        /**
+         * Adds as many incoming transactions as given.
+         * @param incomingTransactions
+         */
+        public void addIncomingTransactions(long incomingTransactions) {
+            this.incomingTransactions += incomingTransactions;
+        } // addIncomingTransactions
+
+        /**
+         * Adds as many bytes for incoming transaction requests as given.
+         * @param incomingTransactionRequestSize
+         */
+        public void addIncomingTransactionRequestSize(long incomingTransactionRequestSize) {
+            this.incomingTransactionRequestSize += incomingTransactionRequestSize;
+        } // addIncomingTransactionRequestSize
+
+        /**
+         * Adds as many bytes for incoming transaction responses as given.
+         * @param incomingTransactionResponseSize
+         */
+        public void addIncomingTransactionResponseSize(long incomingTransactionResponseSize) {
+            this.incomingTransactionResponseSize += incomingTransactionResponseSize;
+        } // addIncomingTransactionResponseSize
+
+        /**
+         * Adds as many incoming transaction errors as given.
+         * @param incomingTransactionErrors
+         */
+        public void addIncomingTransactionErrors(long incomingTransactionErrors) {
+            this.incomingTransactionErrors += incomingTransactionErrors;
+        } // addIncomingTransactionErrors
+        
+        /**
+         * Adds as many service milliseconds as given.
+         * @param serviceTime
+         */
+        public void addServiceTime(double serviceTime) {
+            this.outgoingTransactions++;
+            this.serviceTime += serviceTime;
+        } // addIncomingTransactionErrors
+
+        /**
+         * Merges given metrics with these ones.
+         * @param metrics
+         */
+        public void merge(Metrics metrics) {
+            incomingTransactions += metrics.incomingTransactions;
+            incomingTransactionRequestSize += metrics.incomingTransactionRequestSize;
+            incomingTransactionResponseSize += metrics.incomingTransactionResponseSize;
+            incomingTransactionErrors += metrics.incomingTransactionErrors;
+            outgoingTransactions += metrics.outgoingTransactions;
+            serviceTime += metrics.serviceTime;
+        } // merge
+        
+        /**
+         * Gets the Json string for this metrics.
+         * @return The Json string for this metrics
+         */
+        public String toJsonString() {
+            double avg = (incomingTransactions == 0 ? 0 : serviceTime / outgoingTransactions);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1383/commits/0d04449570086c47f147d87af3fa804bf604b237